LICENSE.SiFive

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Can you just name this file `LICENSE` instead of `LICENSE.SiFive`? There should be no other license files in this repo.